### DISCUSSION TITLE
argocd unable open ui

### DISCUSSION THREAD

**Discussion by mahb2018** [argoproj/argo-cd#6780] [Q&A]
argocd unable open ui，alway display loading font.css

**Comment by jgwest** [comment]
You may wish to provide more details to allow us to provide guidance.

**Comment by tiwarisanjay** [comment]
@jgwest  I am using ArgoCD 2.4.14 . I have integrated the ArgoCD with SSO using OIDC. [ArgoCD Issue](https://files.slack.com/files-pri/T08PSQ7BQ-F04ALS3DY01/image.png)
